recipes/PySimpleGUIWeb/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pysimpleguiweb" %}
+{% set version = "0.35.0" %}
+{% set styledName = "PySimpleGUIWeb" %}
```

Comment:
FYI, you don't need to use `styledName` - you can have `name = "PySimpleGUIWeb"` and use `name|lower` when needed.